Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,9 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+function conversion(origin) {
```

Comment:
Conversion to what? From what?

Reply:
I amended the name of the function from "conversion" to "convertStringsToUppercases", which now makes sense!